### DISCUSSION TITLE
(UK) Getting a Champion to do a short talk at an internal hackathon

### DISCUSSION THREAD

**Discussion by aj-he** [Green-Software-Foundation/discussions#144] [Green Software Champions]
Hi, As part of an internal hackathon at Historic England with the theme "Road to NetZero", I'd like to see if there is a champion in the UK who would be willing to do a short talk to a group of approximately 20 developers to introduce Green Software Development?

I have struggled to message via the social media options due to restrictions on direct messaging if someone is not following you back!

This would be on the 21st May 2024 and could be in person in Swindon or virtually. Thanks!

**Comment by NAMRATA-WOKE** [comment]
@aj-he I think we can help you out. Can you please contact me at namrata@greensoftware.foundation?

**Reply by atg-abhishek** [reply]
@NAMRATA-WOKE I could potentially help with this - please feel free to email me at my GSF address. Thanks! cc @aj-he

**Reply by NAMRATA-WOKE** [reply]
Wonderful. Making the intro!

**Comment by atg-abhishek** [comment]
Thank you!

Abhishek Gupta ( @atg_abhishek ( https://twitter.com/atg_abhishek ) )
Founder and Principal Researcher, Montreal AI Ethics Institute ( https://montrealethics.ai/ )
Director, Responsible AI, Boston Consulting Group (BCG) ( https://www.bcg.com/about/people/experts/abhishek-gupta )
Fellow, Augmented Collective Intelligence, BCG Henderson Institute ( https://bcghendersoninstitute.com/contributors/abhishek-gupta/ )
Author, State of AI Ethics Report ( https://montrealethics.ai/state ) and AI Ethics Brief ( https://brief.montrealethics.ai/ )
Chair, Standards Working Group, Green Software Foundation ( https://greensoftware.foundation/ )
Learn more about my work here ( https://atg-abhishek.github.io/ ). You can now support my work by buying me a coffee ( https://buymeacoff.ee/abhishekgupta ) !
